decidim-core/app/packs/src/decidim/index.js

```diff
@@ -198,7 +198,7 @@ const initializer = (element = document) => {
   // Initialize data-toggles
   element.querySelectorAll("[data-toggle]").forEach((elem) => createToggle(elem))
 
-  element.querySelectorAll("[data-remote-tooltip]").forEach((elem) => fetchRemoteTooltip(elem))
+  // element.querySelectorAll("[data-remote-tooltip]").forEach((elem) => fetchRemoteTooltip(elem))
```

Comment:
```suggestion
  // Temporary disabling this feature because we have a poor performance. See https://github.com/decidim/decidim/issues/14431 
  // element.querySelectorAll("[data-remote-tooltip]").forEach((elem) => fetchRemoteTooltip(elem))
```